bin/genesis

```diff
@@ -997,6 +1082,25 @@ create_bosh_init_environment() {
 EOF
 	done
 
+	export VAULT_PREFIX=secret/${site}/$name/${DEPLOYMENT_NAME}
+	cat <<EOF > ${root}/name.yml
+----
+name: ${site}-${name}-${DEPLOYMENT_NAME}
+meta:
+  vault_prefix: ${VAULT_PREFIX}
+EOF
+
+	if [[ -d ${DEPLOYMENT_ROOT}/.env_hooks ]]; then
+		pushd ${DEPLOYMENT_ROOT}/.env_hooks
```

Comment:
Is it important to the environment hooks that `$PWD` be in the hooks directory?  or does it make more sense to be in `$DEPLOYMENT_ROOT`?

Also, do the hook scripts have access to vars like `$DEPLOYMENT_SITE` and `$DEPLOYMENT_ENV`?